apps/dashboard/components/dashboard/command-menu.tsx

```diff
@@ -68,9 +68,9 @@ export function CommandMenu() {
 const DiscordCommand: React.FC = () => {
   const router = useRouter();
   return (
-    <CommandItem onSelect={() => router.push("/discord")}>
+    <CommandItem onSelect={() => router.push("https://unkey.com/discord")}>
       <svg
-        className="w-4 h-4 mr-2"
+        className="w-4 h-4 mr-2 text-zinc-500 hover:text-white dark:text-zinc-400 dark:hover:text-white transition-colors duration-200"
```

Comment:
I think this should be enough to inherit the correct colors
```suggestion
        className="w-4 h-4 mr-2 text-zinc-500 hover:text-white dark:text-zinc-400 dark:hover:text-white transition-colors duration-200"
        className="w-4 h-4 mr-2 fill-current"
```